.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+### Suggested package name(s)
+<!-- Name suggested package(s) -->
+
+### Alternative to
+<!-- Name popular package(s) the suggested one is an alternative to -->
+
+### Why is the suggested package better?
+<!-- Please explain here -->
+
+### Checklist
+- [ ] Package has sufficient overlap in functionality to act as a replacement.
+- [ ] Package is a better alternative to what is already suggested for this category, or the category is new.
+- [ ] Package is not deprecated and actively maintained (the last release was published less than 6 months ago).
+- [ ] Package has at least 1000 weekly downloads on NPM.
+- [ ] Package has 100+ stars on GitHub.
+- [ ] Package has 5+ dependents on [GitHub](http://bit.ly/gh-depends) or/and NPM.
+- [ ] Package has README and LICENSE.
+- [ ] Package has tests.
```

Comment:
I think that extending these requirements might make it a little cumbersome for folks to submit recommendations. I'm wondering whether we can make some of these checks automated (e.g. check for dependents)? 

I've found that from time to time people forget to fill these out, and even if they do, I have to do a manual check to confirm these. Given the number of recommendations we receive, it might be quicker to just use a github action to auto-fill these?

Reply:
It would also be a good idea to store these as values in our suggestions file. I've found many good packages that don't necessarily have tests, so this shouldn't be a veto criterion, but a ranking factor.

Reply:
I just looking for some polite way to reject suggestions about packages that don't have a community yet. I mean it feels strange to me to add a package that doesn't have depends because that means that the author hasn't done a lot of PR and probably the project might become abandoned soon.

Reply:
Regarding the automation — probably we could implement something like that, but I think it might be a bit difficult to write the test that will check only the packages that were added in the PR. Anyway, I would start with a manual check and implement automation later if we feel that we need that.

Reply:
Once we'll have auto-checks, we could drop these lines from the manual checklist.

Reply:
> I just looking for some polite way to reject suggestions about packages that don't have a community yet.

Isn't the best we can do, but this is how I've framed it in the past —
https://github.com/pastelsky/bundlephobia/issues/249#issuecomment-569576859

Reply:
I guess what I'm saying is, these are all factors in consideration. Not having 100+ stars or tests should not disqualify a package from being added as a suggestion if its genuinely good.

I'm not sure if this will scale, but I want some to retain some human decision making element while adding these. There are already sites (npms.io or npm's own quality indicators on npmjs.org) that quantify these as abstract quality scores.

Reply:
So we both agree that having depents it's an important requirement.
I would keep the requirement in the checklist for now, but replaced it with an auto-check as soon as we'll be able to implement it. 

But If you are sure that we should remove it from the PR template, I will do that.

Reply:
BTW: Removed "has tests" and "has README/LICENCE" items.